tests/FsCheck.Test/Runner.fs

```diff
@@ -202,7 +296,7 @@ module BugReproIssue344 =
             
         // run this on another thread, and abort it manually
         let thread2 = Thread(fun () ->
-            let config = { Config.Quick with Arbitrary = [ typeof<MyGenerators> ]  }
+            let config = { Config.Quick with Arbitrary = [ typeof<MyGenerators> ]; ParallelRunConfig = None  }
```

Comment:
As far as I recall there were several issues (forgot which exactly) with running this scenario in parallel mode. For now It runs ok in parallel. Also note that I left parallel mode as [default](https://github.com/fscheck/FsCheck/pull/351/files/fac6f6b7a1cccb6420bcf3768cc47d66ed24100f#diff-fae3712c738f2860b54d6ce6f880b74aR536)  for now. Just to see how it will behave on CI server, will remove before merge.